docker_config/dockerfiles/Dockerfile.default-cpu

```diff
@@ -1,7 +1,10 @@
 FROM ubuntu:xenial
 MAINTAINER CodaLab Team "codalab.worksheets@gmail.com"
 
+RUN apt-get update && apt-get install -y \
+    software-properties-common
```

Comment:
Why is this change needed? Also, why did we remove python-software-properties in default-gpu but not in this file?

Reply:
The step after this `add-apt-repository` will fail if we don't install software-properties-common.

Reply:
I also added python-software-properties back, thanks for the catch.